torch/optim/adam.py

```diff
@@ -70,7 +73,7 @@ class Adam(Optimizer):
 
     def __init__(self, params, lr=1e-3, betas=(0.9, 0.999), eps=1e-8,
                  weight_decay=0, amsgrad=False, *, foreach: Optional[bool] = None,
-                 maximize: bool = False):
+                 maximize: bool = False, capturable: Optional[bool] = False):
```

Comment:
sure but why is None better than False?

Reply:
I think what @ezyang is asking is whether `capturable` is expected to ever take on a `None` value rather than suggesting to change this to `capturable: Optional[bool] = None`. If not, then the type annotation can just be `capturable: bool = False` since `Optional` refers to permitting `None` rather than referring to having a default argument.